components/molecules/BenefitCard.test.js

```diff
@@ -47,10 +47,64 @@ const activeCanadaPensionPlan = {
   ],
 }
 
+const activeCppTasks = [
```

Comment:
This can just be imported from constants instead of adding it here too. I already updated this test as part of #204